staging/src/k8s.io/apiserver/pkg/registry/generic/registry/store.go

```diff
@@ -1035,13 +1035,15 @@ func (e *Store) DeleteCollection(ctx context.Context, deleteValidation rest.Vali
 			for index := range toProcess {
 				accessor, err := meta.Accessor(items[index])
 				if err != nil {
-					errs <- err
-					return
+					select {
```

Comment:
I don't like this change because it means that some errors can end up unreported.  The buffer is only as big as the number of workers, not the amount of work.

This also reveals another problem with the method: not all errors are reported, not even numbers of errors.

On the practical side, this is a very unlikely error to get.  We care more about the actual deletion errors. below.

Reply:
Adopted error reporting summary suggested below.

Reply:
You cannot use a select here, that drops errors on the floor.